docs/packaging/submitting-a-pull-request.mdx

```diff
@@ -37,25 +42,47 @@ git log -1
 
 ## Creating the Pull Request
 
-In the package folder, run `git push`.
-
-:::note
-
-If you've created your own branch, as recommended, the cli tool will show you a new command to create and push to a remote branch matching the local one. Run this.
-
-:::
-
-Once the commit is successfully pushed, you'll notice that a URL will be provided that will immediately allow you to create a pull request with your changes.
-
-Run `git status` one last time to make sure your branch is clean. If it is:
-
-1. Open the link.
-2. Fill in a summary of your changes (usually the same as the commit message).
-3. Link any relevant issues:
-   - If you want to link this pull request to an existing issue, simply mention it in the PR summary: `The inclusion of <somepackage> fixes #123`.
-   - If you need a change to depend on another change, mention it in the PR summary too: `Depends on #234`.
-4. Double check everything
-5. Create the pull request!
+There are multiple ways to create a Pull Request with GitHub, either from the website, or from the command line.
+
+<Tabs groupId="opening-prs">
+  <TabItem value="website" label="Website">
+    <p>
+    In the package folder, push your local changes to a remote branch:
+
+    <CodeBlock language="bash">
+      git push
+    </CodeBlock>
+    </p>
+    <Admonition type="info">
+      If you've created your own branch, as recommended, the CLI tool will show you a new command to create and push to a remote branch matching the local one. Run this.
+    </Admonition>
+    <p>
+    Once the commit is successfully pushed, you'll notice that a URL will be provided that will immediately allow you to create a pull request with your changes.
+
+    Run `git status` one last time to make sure your branch is clean. If it is:
+
+    1. Open the link.
+    2. Fill in a summary of your changes (usually the same as the commit message).
+    3. Link any relevant issues:
+       - If you want to link this pull request to an existing issue, simply mention it in the PR summary: `The inclusion of <somepackage> fixes #123`.
+       - If you need a change to depend on another change, mention it in the PR summary too: `Depends on #234`.
+    4. Double check everything
+    5. Create the pull request!
+    </p>
+
+  </TabItem>
+  <TabItem value="github-cli" label="GitHub CLI">
+    <p>
+    In the package folder, run the `gh` command:
+    </p>
+    <CodeBlock language="bash">
+      gh pr create
+    </CodeBlock>
+    <p>
+    This will start an interactive session to create your pull request step-by-step.
```

Comment:
Can we add a section something along the lines of:

Note that the text editor that `gh` uses for editing the PR body message is different from the editor used by `git` (i.e. it doesn't respect the setting of `git`). If you wish to change this, consult the [`gh config set`](https://cli.github.com/manual/gh_config_set) command.

I was often annoyed by `gh` opening up `nano` until I discovered this that allowed me change it to `vim`.

Reply:
I had no idea you could do this. That's been a *major* annoyance for me, especially since I use Zellij, which means I have to do an additional key combo so it doesn't handle keyboard combinations meant for Nano. Awesome!

Reply:
Added an info box for this.